jdk17-unit-tests/src/test/java/com/uber/nullaway/jdk17/NullAwayOptionalEmptyTests.java

```diff
@@ -0,0 +1,98 @@
+package com.uber.nullaway.jdk17;
+
+import com.google.errorprone.CompilationTestHelper;
+import com.uber.nullaway.NullAway;
+import java.util.Arrays;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+/** Tests for support of the {@code Optional.isEmpty()} API. This API was introduced in JDK 11. */
+public class NullAwayOptionalEmptyTests {
```

Comment:
I stuck this inside the `jdk17-unit-tests` module for simplicity.  Technically it only requires JDK 11 but I didn't want to create a separate `jdk11-unit-tests` module just for that.